crates/evm/evm.rs

```diff
@@ -242,6 +263,59 @@ pub fn evm_state(store: Store) -> EvmState {
     )
 }
 
+/// Calls the eip4788 beacon block root system call contract
+/// As of the Cancun hard-fork, parent_beacon_block_root needs to be present in the block header.
+pub fn beacon_root_contract_call(
+    state: &mut EvmState,
+    header: &BlockHeader,
+    spec_id: SpecId,
+) -> Result<ExecutionResult, EvmError> {
+    let system_caller_address =
+        RevmAddress::from_slice(&hex::decode("fffffffffffffffffffffffffffffffffffffffe").unwrap());
+    let beacon_root_contract_address = RevmTxKind::Call(RevmAddress::from_slice(
+        &hex::decode("000F3df6D732807Ef1319fB7B8bB8522d0Beac02").unwrap(),
+    ));
```

Comment:
We could make this address into a constant (with lazy static) to avoid creating it for each execution